src/topology/metric_space/basic.lean

```diff
@@ -2881,3 +2881,66 @@ instance metric_space_quot {α : Type u} [pseudo_metric_space α] :
     λxc yc zc, quotient.induction_on₃ xc yc zc (λx y z, pseudo_metric_space.dist_triangle _ _ _) }
 
 end eq_rel
+
+/-! ### `additive`, `multiplicative` -/
```

Comment:
A brief comment indicating that the structure is inherited in the obvious way would be good here and below all the other headings.